### ISSUE TITLE
Make method async code fix on inner method applies async on wrong method

### ISSUE THREAD

**Issue by twsouthwick** [dotnet/roslyn#19437]
**Version Used**: 
VS 2017.2

**Steps to Reproduce**:

1. Use the following code
2. Invoke code fixer on line with `await Task.CompletedTask`

```csharp
        public void OuterMethod()
        {
            Task InnerMethod()
            {
                await Task.CompletedTask;
            }
        }
```

**Expected Behavior**:

```csharp
        public void OuterMethod()
        {
            async Task InnerMethod()
            {
                await Task.CompletedTask;
            }
        }
```

**Actual Behavior**:

```csharp
        public async Task OuterMethod()
        {
            Task InnerMethod()
            {
                await Task.CompletedTask;
            }
        }
```

**Comment by jcouv** [comment]
FYI @agocke (relates to local functions)

**Comment by Pilchie** [comment]
I have a vague feeling that this might be a dupe - @sharwell can you take a look?

**Comment by sharwell** [comment]
This is a duplicate of #14133. The fix has already been merged, but I missed the cutoff for 15.3 preview 1. It will be in whatever the next release after that is.
